src/gc/impl/conservative/gc.d

```diff
@@ -1820,6 +1835,8 @@ struct Gcx
         if (pool)
         {
             pool.initialize(npages, isLargeObject);
+            if (collectInProgress)
+                pool.mark.setAll();
```

Comment:
This is definitely not a problem for the concurrency/bit sharing, as this memory will not ever even be seen by the forked mark process as it has an old snapshot of the memory. This pool will not be marked until the next collection.